distribution/docs/src/main/resources/content/_metadataAttributes/NITF_Segment_Image-table.adoc

```diff
@@ -112,11 +112,21 @@
 |true
 
 |ImageSegment.TGTID
-|trimmed text value
+|none
 |ext.nitf.image.target-identifier
 |String
 |true
 
+|ImageSegment.TGTID
+|The last 2 characters of ImageSegment.TGTID correspond to the country code. If these characters are not spaces then null. Else,
+
+2char FIPS -> 3char ISO3
+
+If conversion fails, then null. NOTE: Other system plugins could populate the country code based on location.
+|<<location.country-code,location.country-code>>
+|String
+|true
```

Comment:
![image](https://user-images.githubusercontent.com/8041246/43424034-eb59e9d2-9402-11e8-9877-07a99d04e7d9.png)
I confirmed that this hyperlinks jumps to the right place.